src/Kmd.Logic.Audit.Client.SerilogSeq/SerilogSeqAuditClientConfigurationExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Net.Http;
+using Serilog;
+
+namespace Kmd.Logic.Audit.Client.SerilogSeq
+{
+    public static class SerilogSeqAuditClientConfigurationExtensions
+    {
+        public static LoggerConfiguration DefaultConfiguration(this SerilogSeqAuditClientConfiguration config, HttpMessageHandler messageHandler = null)
+        {
+            if (config == null)
+            {
+                throw new ArgumentNullException(nameof(config));
+            }
+
+            var configBuilder = new LoggerConfiguration()
+                .AuditTo.Seq($"{config.ServerUrl}", apiKey: config.ApiKey, messageHandler: messageHandler, compact: true);
```

Comment:
I can see you copied this, but it looks strange, I wonder why we do this?!

```suggestion
                .AuditTo.Seq(config.ServerUrl, apiKey: config.ApiKey, messageHandler: messageHandler, compact: true);
```

Reply:
Done